CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# How to become a contributor and submit your own code
+
+We'd love to accept your issues and (especially) patches! If you have found a
+bug you can submit an issue describing it and one of the repo maintainers will
+get back to you. Alternately you can submit a PR and the sections below outline
+the process.
+
+## Contributing a Patch
+
+1. Fork the desired repo, develop and test your code changes.
+1. Ensure that your code has an appropriate set of unit tests which all pass.
+1. Break your changes in to a series of logical self-contained commits.
+1. Include release notes files when appropriate following the guidelines
+   in [Neptune](https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-contributor.rst).
+1. Submit your pull request.
```

Comment:
The numbers are all 1s.

Reply:
Yeah it means I don't need to change them if I change the order. They render ok.

Reply:
Wow, I didn't know that :+1:

Reply:
I was looking at a few other contributions pages for ideas and saw it :)